napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -166,31 +162,29 @@ def test_drag_in_add_mode(create_known_points_layer_2d):
 
     # Add point at location where non exists
     layer.mode = 'add'
-    layer.mouse_pan = True
 
     # Simulate click
-    event = ReadOnlyWrapper(
-        Event(type='mouse_press', position=known_non_point)
+    event = read_only_event(
+        type='mouse_press', position=known_non_point, pos=np.array([0, 0])
     )
     mouse_press_callbacks(layer, event)
 
     known_non_point_end = [40, 60]
 
     # Simulate drag end
-    event = ReadOnlyWrapper(
-        Event(
-            type='mouse_move', is_dragging=True, position=known_non_point_end
-        )
+    event = read_only_event(
+        type='mouse_move',
+        is_dragging=True,
+        position=known_non_point_end,
+        pos=np.array([4, 4]),
     )
     mouse_move_callbacks(layer, event)
 
     # Simulate release
-    event = ReadOnlyWrapper(
-        Event(
-            type='mouse_release',
-            position=known_non_point_end,
-            pos=np.array([4, 4]),
-        )
+    event = read_only_event(
+        type='mouse_release',
+        position=known_non_point_end,
+        pos=np.array([4, 4]),
```

Comment:
The add/not-add-point behaviour is already tested here. Testing whether the camera moves is quite a bit harder cause we need a fully working vispy canvas, and I don't seem to be able to get that to work. I'm using `make_napari_viewer` and then checking if `viewer.window.qt_viewer.camera.center` changes before and after the callback. For some reason, this never changes, despite the fact that when I'm actually testing manually, this value is changed as appropriate. Any suggestions? Otherwise I suggest we get this in without adding an extra test, since it's a small bug fix.